.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,163 @@
+name: Publish Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            environment:
+                type: environment
+                default: "Beta"
+            version:
+                type: string
+                description: "Version (semver) for the release."
+                default: "0.0.1"
+                required: true
+            build:
```

Comment:
I'm not sure if it makes sense to include a build number in the release action. The build number was only ever intended to be used internally, as a way to distinguish iterations between releases. When we create a release, the version number gets a bump (i.e. v4.0.0 -> v4.0.1) which resets the build number, so it's always going to be `0`. And there is not going to be any releases between v4.0.1 and v4.0.2. Conversely, for our continuous builds (which are not releases) the version number will stay the same, and the build number should increase with every build. Does that make sense?

Reply:
Yes it does! I think it doesn't harm to let this be a parameter since it's actively used for the Firefox extension publishing (and Chrome, once uploaded), and just always let it be the default of 0. If you think it's confusing, I can just remove it entirely, though.

Reply:
I guess you're right that it doesn't hurt, especially if it's optional. So let's keep it in for now.